packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -73,7 +73,7 @@ function validateCategories(categories) {
 	if (!categories) return null;
 	if ((categories.length !== new Set(categories).size)) throw new Error('Repeated categories are not allowed');
 	categories.forEach(category => {
-		if (!allPossibleCategories.includes(category)) throw new Error(`${category} is not a valid category. Please make sure that the category name is lowercase. Valid Categories are: \n${allPossibleCategories}\n`);
+		if (!allPossibleCategories.map(category => { return category.name; }).includes(category)) throw new Error(`${category.name} is not a valid category. Please make sure that the category name is lowercase. Valid categories are: \n${allPossibleCategories}\n`);
```

Comment:
> throw new Error(`${category.name}

In fact it should be "category" here, not "category.name", isn't it? It doesn't look like you have tested or even read your code at all. I made a mistake by suggesting this change, but you need to be responsible for what you're doing and not rely on reviewers.

Reply:
tested and fixed now